src/compiler/checker.ts

```
@@ -6662,46 +6665,94 @@ namespace ts {
+         */
+        function checkClassPropertyAccess(node: PropertyAccessExpression | QualifiedName, left: Expression | QualifiedName, type: Type, prop: Symbol): boolean {
             let flags = getDeclarationFlagsFromSymbol(prop);
-            // Public properties are always accessible
+            let declaringClass: InterfaceType;
+
+            if (left.kind === SyntaxKind.SuperKeyword) {
+                let errorNode = (<PropertyAccessExpression>node).name ?
+                    (<PropertyAccessExpression>node).name :
+                    (<QualifiedName>node).right;
+
+                // TS 1.0 spec (April 2014): 4.8.2
+                // - In a constructor, instance member function, instance member accessor, or
+                //   instance member variable initializer where this references a derived class instance,
+                //   a super property access is permitted and must specify a public instance member function of the base class.
+                // - In a static member function or static member accessor
+                //   where this references the constructor function object of a derived class,
+                //   a super property access is permitted and must specify a public static member function of the base class.
+                if (getDeclarationKindFromSymbol(prop) !== SyntaxKind.MethodDeclaration) {
+                    // `prop` refers to a *property* declared in the super class
+                    // rather than a *method*, so it does not satisfy the above criteria.
+
+                    error(errorNode, Diagnostics.Only_public_and_protected_methods_of_the_base_class_are_accessible_via_the_super_keyword);
+                    return false;
+                }
+
+                // In a super call, the methods cannot be accessed if the method is abstract.
+                // Note that a member cannot be private and abstract -- this is checked elsewhere.

```

Yeah I agree, it isn't really relevant. I was hoping to indicate that this check can mask other errors. Is this better, or should I just remove the second line altogether?

```ts
                    // A method cannot be accessed in a super call if the method is abstract.
                    // This error could mask a private property access error. But, a member 
                    // cannot simultaneously be private and abstract, so this will trigger an
                    // additional error elsewhere.

                    // ...
```